Rakefile

```diff
@@ -1,6 +1,24 @@
 require 'airbrake/tasks'
 require_relative "email_alert_service/environment"
+require 'rspec/core/rake_task'
 
 # The airbrake tasks rely on the Railsy "environment" task
 # which is covered by the above require.
 task :environment do; end
+
+namespace :message_queue do
+  desc "Run worker to consume messages from rabbitmq"
+  task :consumer => [:environment] do
+    GovukMessageQueueConsumer::Consumer.new(
+      queue_name: "email_alert_service",
+      exchange: "published_documents",
+      processor: MessageProcessor.new,
+      routing_key: "*.major"
```

Comment:
Any reason why this `routing_key` is different from the original? https://github.com/alphagov/email-alert-service/commit/c147d9f73472006d823020515449bdb212e34e3a